server/data/nomisClientBuilder.js

```diff
@@ -102,10 +92,6 @@ module.exports = context => {
       const path = `${apiUrl}api/bookings/${bookingId}/mainOffence`
       return nomisClientGet({ path })
     },
-    getMainOffences(bookingIds) {
-      const path = `${apiUrl}api/bookings/mainOffence`
-      return nomisClientPost({ path, body: bookingIds })
-    },
```

Comment:
These are the 3 endpoint calls that have bee moved to prisoner search, getSentenceDatesForOffenders and getMainOffences both now use the same prisoner search endpoint as the data from both is returned from it